tailwind.config.js

```diff
@@ -1,12 +1,36 @@
 /** @type {import('tailwindcss').Config} */
 export default {
-  content: [
-    "./index.html",
-    "./src/**/*.{js,ts,jsx,tsx}",
-  ],
+  content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
   theme: {
     extend: {},
+    colors: {
+      //Background
+      dkGeneralBgC: "#1A1A1A",
+      dkPrimaryBgC: "#2A2133",
+      dSecondaryBgC: "#D1A5FF",
```

Comment:
dkSecondaryBgC  or dSecondaryBgC is correct?

Reply:
done